app/models/document.rb

```diff
@@ -25,6 +25,26 @@ class Document < ApplicationRecord
   validates_inclusion_of :review_state, in: REVIEW_STATES
   validates_inclusion_of :update_type, in: %w[major minor], allow_nil: true
 
+  scope :draft, -> {
```

Comment:
It's curious that this is (hopefully) the same logic as in UserFacingState. I wonder if there's a way to combine the two?

Reply:
Yeah, it'd be nice to combine them. I'm not sure of any way to do that though since scope is in the context of queries whereas UserFacingState operates on attributes of a model.